library/src/scripts/forms/buttonStyles.ts

```diff
@@ -50,6 +51,9 @@ export const buttonGlobalVariables = useThemeCache(() => {
         minHeight: formElVars.sizing.height,
         minWidth: 104,
         compactHeight: 24,
+        mobile: {
+            minHeight: 24,
+        },
```

Comment:
Could we not use a new variable here and just use the `compactHeight`? I think that's the purpose.

Reply:
Fixed - use compactHeight